packages/playground/src/weblets/micro_vm.vue

```diff
@@ -245,7 +244,6 @@ const mycelium = ref(true);
 const wireguard = ref(false);
```

Comment:
While we enhance the network component, I would recommend refactoring the weblets by removing all network consents

it could be simply 

```suggestion
function networks(){
    return {
        planetary, mycelium, publicIpv4, publicIpv6
    } as ref()
}

const networks = createNetworks()
const vm = await deployVM(grid!, {
      planetary: networks.planetary.value,
      mycelium: networks.mycelium.value,
      publicIpv4: networks.ipv4.value,
      publicIpv6: networks.ipv6.value,
})
```

PS: Me and @AhmedHanafy725 agreed [in](https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3287) on adding all networks to all solutions and enabling the `Mycelium` 
 option by default

please check the Jenkins solution

![image](https://github.com/user-attachments/assets/01c51ce4-d005-4cd3-9d03-84d1b4026778)